src/dialogs/config-flow/dialog-data-entry-flow.ts

```diff
@@ -322,7 +322,7 @@ class DataEntryFlowDialog extends LitElement {
       haStyleDialog,
       css`
         ha-paper-dialog {
-          max-width: 500px;
+          max-width: 600px;
```

Comment:
I know why you did this, but I think for a future PR we should just convert the grid to a single list. Left side name/manufactuer/model, right side area. I'll open an issue for it.